Mergin/plugin.py

```diff
@@ -165,6 +166,12 @@ def initGui(self):
             self.iface.addCustomActionForLayerType(self.action_show_changes, "", QgsMapLayer.VectorLayer, True)
             self.action_show_changes.triggered.connect(self.view_local_changes)
 
+            # add layer context menu action for downloading vector tiles
+            self.action_export_mbtiles = QAction("Make available offline…", self.iface.mainWindow())
```

Comment:
@alexbruy can we have this action only for vector tiles layers? Other layers are not supported anyways